Dockerfile.cli

```
@@ -0,0 +1,35 @@
+FROM node:18-bullseye as base
+FROM base as deps-files
+WORKDIR /app
+COPY --link yarn.lock package.json .
+COPY --link packages packages
+RUN find . -type f -not -iname "yarn.lock" -not -iname "package.json" -delete && find . -type l -delete && find . -type d -empty -delete 
```

instead of copying the entire source tree and then deleting everything except package.json

```suggestion
COPY --link packages/*/package.json packages
```